tests/test_baseclient.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""Unit tests for RevAiApiClient"""
+
+import pytest
+import re
+import json
+from requests.exceptions import HTTPError
+from src.rev_ai.apiclient import RevAiAPIClient
+from src.rev_ai import __version__
+from src.rev_ai.baseclient import BaseClient
+from tests.helpers.errors import get_error_test_cases
+
+TOKEN = "token"
+
+
+class TestBaseClient:
+    def test_constructor_with_success(self):
+        client = BaseClient(TOKEN)
+
+        headers = client.default_headers
+
+        assert headers.get(
+            'User-Agent') == 'RevAi-PythonSDK/{}'.format(__version__)
+        assert headers.get('Authorization') == 'Bearer {}'.format(TOKEN)
```

Comment:
In my opinion we should be using string literals in the assertions where possible. String literals are less prone to mistakes.